_posts/2022-12-06-poetry-ai.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "AI and poetry meet Trino"
+author: Martin Traverso, Marcos Traverso
+excerpt_separator: <!--more-->
+image: /assets/trino-light.png
+---
+
+Sometimes working in the depth of the query engine core, the planner, the
+optimizer, or some other tricky problem or code, you run out of steam and need a
+distraction. Your brain will quietly and in the background still work on the
+problem, while you get to read or do something interesting.
+
+That's when we found out more about GPT-3 and the recent improvements on the
+OpenAI project. So we had to try.
+
+<!--more-->
+
+Its pretty amazing that you can feed the AI a request just like we did:
+
+```
+Write a short rhyming poem explaining Trino in easy but accurate terms
+```
+
+And after a few trials, the best one produced by GPT-3 was this:
+
+```
+Trino is a tech platform,
+That makes data integration a breeze,
+It's designed to help your business grow,
+And optimise your systems with ease.
+It links together different sources,
+Both big and small,
+Gathers data of different types,
+Helping you make decisions for all.
+So Trino is an integration tool,
+That connects data for you,
+It'll ease the burden of data sync,
+Giving you more time to do what you do.
+```
+
+Cool, huh? I am sure we could not have done better even after a long time
+trying. Have a look at another question:
```

Comment:
This might be the bachelor's in English talking, but we definitely could've done better, and I'd probably rephrase this to be a little less grandiose of a claim...